messytables/excel.py

```diff
@@ -22,32 +24,47 @@
     4: IntegerType()
 }
 
-
 class XLSTableSet(TableSet):
     """An excel workbook wrapper object.
     """
 
     def __init__(self, fileobj=None, filename=None,
-                 window=None, encoding=None):
+                 window=None, encoding=None, with_formatting_info=True):
+        def get_workbook():
```

Comment:
i believe this function definitioin should go after the docstring (that starts on line 44, below).